src/app/components/signup/signup.component.html

```diff
@@ -1,19 +1,82 @@
-@if (errorMessage) {
-<div>{{ errorMessage }}</div>
-}
+<div
+  class="bg-cover bg-center h-screen"
+  style="background-image: url('../../assets/backgrounds/travel.jpeg')"
+>
+  <div
+    class="bg-black bg-opacity-30 h-full w-full flex flex-col items-center justify-between"
+  >
+    <div class="h-1/3 flex justify-center items-center">
+      <img
+        class="size-40 px-4"
+        src="../../assets/logos/destigo-high-resolution-logo-transparent.svg"
+        alt="image description"
+      />
+    </div>
 
-<form [formGroup]="form" (ngSubmit)="onSubmit()">
-  <div>
-    <label for="name">Name</label>
-    <input type="text" id="username" formControlName="username" />
-  </div>
-  <div>
-    <label for="email">Email</label>
-    <input type="email" id="email" formControlName="email" />
-  </div>
-  <div>
-    <label for="password">Password</label>
-    <input type="password" id="password" formControlName="password" />
+    <div
+      class="bg-white h-2/3 w-full rounded-t-badge flex flex-col items-center justify-around p-12"
+    >
+      <h1 class="text-3xl text-black font-bold">New Account</h1>
+      @if (errorMessage) {
+      <div>{{ errorMessage }}</div>
+      }
+
+      <form
+        [formGroup]="form"
+        (ngSubmit)="onSubmit()"
+        class="w-full flex flex-col gap-6"
+      >
+        <div class="flex flex-col">
+          <label for="name" class="text-dark-gray">Username:</label>
+          <nz-input-group
+            class="border bg-transparent border-transparent border-b-coral"
+            nzPrefixIcon="user"
+          >
+            <input
+              type="text"
+              nz-input
+              formControlName="userName"
+              placeholder="Username"
+            />
+          </nz-input-group>
+        </div>
+        <div class="flex flex-col">
+          <label for="email" class="text-dark-gray">Email:</label>
+          <nz-input-group
+            class="border bg-transparent border-transparent border-b-coral"
+            nzPrefixIcon="user"
+          >
+            <input
+              type="text"
+              nz-input
+              formControlName="email"
+              placeholder="Email"
+            />
+          </nz-input-group>
+        </div>
+        <div class="flex flex-col">
+          <label for="password" class="text-dark-gray">Password:</label>
+          <nz-input-group
+            class="border bg-transparent border-transparent border-b-coral"
+            nzPrefixIcon="lock"
+          >
+            <input
+              type="password"
+              nz-input
+              formControlName="password"
+              placeholder="Password"
+            />
+          </nz-input-group>
+        </div>
+        <div class="flex w-full justify-center">
+          <button
+            type="submit"
+            class="bg-coral flex justify-center p-4 items-center text-lg text-white w-3/4 rounded-full"
+          >
+            <p>Sign up</p>
+          </button>
```

Comment:
like here? You add the button, but then you have a separate Sign Up button component?